### PR TITLE
OADP Blog init + odf-oadp blog

### PR DESCRIPTION
**Pull request by kaovilai**
Close [OADP-285](https://issues.redhat.com/browse/OADP-285)
View rich markdown (ie. with pictures) at https://github.com/kaovilai/oadp-operator/blob/blog/blogs/oadp-odf/oadp-blog.md
Wherever this blog gets published to.. [tig.pw/oadp-blog](https://tig.pw/oadp-blog) shall redirect there.
GDoc https://docs.google.com/document/d/1jFM4QATN_DAG6W5pga00LdcSwYBtKdURz4IbRXGrlPY/edit?usp=sharing

### CODE REVIEW

File: blogs/oadp-odf/oadp-blog.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
+# Guide: back up and restore persistent workloads on OpenShift using OADP and ODF
```
Comment (dymurray) [inline blogs/oadp-odf/oadp-blog.md:1]: NIT:

>Backup and Restore Stateful Applications on OpenShift with OADP and ODF

File: blogs/oadp-odf/oadp-blog.md [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+-   Installation
+-   Application Overview
+-   Backup/Restore of app showing CSI functionality
+-   Upcoming features (Data Mover)
+-->
+
+OpenShift API for Data Protection (OADP) enables backup, restore, and disaster recovery of applications on an OpenShift cluster. Data that can be protected with OADP include k8s resource objects, persistent volumes, and internal images.
+The OpenShift API for Data Protection (OADP) is designed to protect Application Workloads on a single OpenShift cluster.
+
+Red Hat® OpenShift® Data Foundation—previously Red Hat OpenShift Container Storage—is software-defined storage for containers. Engineered as the data and storage services platform for Red Hat OpenShift, Red Hat OpenShift Data Foundation helps teams develop and deploy applications quickly and efficiently across clouds.
```
Comment (dymurray) [inline blogs/oadp-odf/oadp-blog.md:15]: I would remove "previously OCS..."

File: blogs/oadp-odf/oadp-blog.md [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+- Application deployment
+- Application protection
+- A disaster scenario
+- Application recovery from disaster
+## Pre-requisites
+- Terminal environment
+  - Your terminal has the following commands
+    - [oc](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/cli_tools/openshift-cli-oc) binary
+    - [git](https://git-scm.com/downloads) binary
+  - Alternatively enter prepared environment in your terminal with `docker run -it ghcr.io/kaovilai/oadp-cli:v1.0.1 bash`
```
Comment (dymurray) [inline blogs/oadp-odf/oadp-blog.md:31]: This is cool. Maybe include `velero` binary too.

TBH you can also likely just assume someone has the tools installed too.

File: blogs/oadp-odf/oadp-blog.md [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+## Deploying an application
+Since we are using OpenShift Data Foundation, we can use common application definition across cloud providers regardless of available storage class.
+
+Clone our demo apps repository and enter the cloned repository.
+```sh
+git clone https://github.com/kaovilai/mig-demo-apps --single-branch -b oadp-blog-rocketchat
+cd mig-demo-apps
+```
+
+Apply rocket chat manifests.
```
Comment (dymurray) [inline blogs/oadp-odf/oadp-blog.md:102]: I like the rocket chat usage

File: blogs/oadp-odf/oadp-blog.md (hunk 3 above)
Comment (kaovilai) [reply blogs/oadp-odf/oadp-blog.md:31]: I was trying to get away with less. I actually in the cli container include a velero alias which uses velero command from velero server on cluster to
- save on downloads
- eliminate version mismatch between client cli/server

I can add an alias command as a note for velero.

**Comment by kaovilai** [comment]
Lmao commit was 26 days ago.. CI bot replying just now?

**Comment by kaovilai** [comment]
This has published at https://cloud.redhat.com/blog/how-to-backup-and-restore-stateful-applications-on-openshift-using-oadp-and-odf

**Pull request merged**